src/app/services/storage.service.ts

```diff
@@ -105,8 +105,14 @@ export class StorageService {
 
   // Check to see if every VDEV has the same capacity. Best practices dictate every vdev should be uniform
   isMixedVdevCapacity(allVdevCapacities: Set<number>): boolean {
+    const max = Math.max(...allVdevCapacities);
+    const min = Math.min(...allVdevCapacities);
+    const diffPercentage = 100 - ((min / max) * 100);
     const diff = Math.max(...allVdevCapacities) - Math.min(...allVdevCapacities);
-    return allVdevCapacities.size > 1 && diff >= GiB * 2;
+
+    const isDiffNoticable = diffPercentage >= 5 && diff >= GiB * 2;
```

Comment:
Since the two factors (swap and different number of metaslabs) may coexist, I would lean towards `2GB + 5%` rather than `2GB and 5%` here.  But area when it would make the difference is already pretty shady (very small disks), so I don't insist.